ci/fixtures.json

```diff
@@ -0,0 +1,41 @@
+[
+{
+  "model": "auth.user",
+  "pk": 1,
+  "fields": {
+    "password": "pbkdf2_sha256$150000$EoxUbjPQTZBy$OyNZocrY9aewIVmB2nbnCoZVfDh9PC9gBOtXG4Nb8t8=",
```

Comment:
Is this the hashed password? If so we should add a `Development` entry in the `README` where we say the actual password.